articles/sentinel/dns-normalization-schema.md

```diff
@@ -118,14 +118,16 @@ Event fields are common to all schemas, and describe the activity itself and the
 | **EventResult** | Mandatory | Enumerated | `Success` | One of the following values: **Success**, **Partial**, **Failure**, **NA** (Not Applicable).<br> <br>The value may be provided in the source record using different terms, which should be normalized to these values. Alternatively, the source may provide only the [EventResultDetails](#eventresultdetails) field, which should be analyzed to derive the EventResult value.<br> <br>If this record represents a request and not a response, set to **NA**. |
 | <a name=eventresultdetails></a>**EventResultDetails** | Mandatory | Alias | `NXDOMAIN` | Reason or details for the result reported in the **_EventResult_** field. Aliases the [ResponseCodeName](#responsecodename) field.|
 | **EventOriginalUid** | Optional | String | | A unique ID of the original record, if provided by the source. |
-| **EventProduct** | Mandatory | String | `DNS Server` | The product generating the event. This field may not be available in the source record, in which case it should be set by the parser. |
+| **EventOriginalType**   | Optional    | String     |   The original event type or ID, if provided by the source. <br><br>Example: `lookup`|
```

Comment:
@oshezaf for some reason we never moved this article from 5 columns to 4 (should probably do that after this set of updates). therefore this row should have the following syntax:

`| **EventOriginalType**   | Optional    | String    | `lookup` |   The original event type or ID, if provided by the source. |`

Reply:
Yup. I modified others, but missed this one. I did keep DvcId, which I also added here, intentionally to keep table width: the example is very long, as I figured we need to update this to 4 columns anyways.